path.go

```diff
@@ -387,26 +387,12 @@ func (b *PathBuilder) Recursive(selector string) *PathBuilder {
 	return b
 }
 
-func (b *PathBuilder) containsReservedPathCharacters(path string) bool {
-	if strings.Contains(path, ".") {
-		return true
-	}
-	if strings.Contains(path, "*") {
-		return true
-	}
-	return false
-}
-
-func (b *PathBuilder) enclosedSingleQuote(name string) bool {
-	return strings.HasPrefix(name, "'") && strings.HasSuffix(name, "'")
-}
-
-func (b *PathBuilder) normalizeSelectorName(name string) string {
-	if b.enclosedSingleQuote(name) {
+func normalizeSelectorName(name string) string {
```

Comment:
Can it be modified to minimize the difference ?
If necessary, I would like to do the refactoring separately.

Reply:
Updated